.github/workflows/e2e-test.yml

```
@@ -0,0 +1,46 @@
+name: Playwright E2E Tests
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)